docs/userGuide/usingPlugins.md

```diff
@@ -76,7 +76,7 @@ Plugins come as `.js` files. To install an external plugin, simply put it in the
 
 ## Writing Plugins
 
-You may also write your own plugins! Refer [here]({{baseUrl}}/devdocs/devGuide/writingPlugins.html) for the available interfaces to do so.
+You may also write your own plugins! Refer [here](https://markbind.org/devdocs/devGuide/writingPlugins.html) for the available interfaces to do so.
```

Comment:
Does this need to be the absolute URL?

Reply:
this is a little tricky (related to the unresolved pr https://github.com/MarkBind/markbind/pull/1211)

`/devdocs` is the base url of the dg -- so when building the site using either `ug-site.json` or `site.json` both would report false negatives (the link checking needs to strip off the base url first)

since the dev guide is always updated with the latest things I just redirected it to point to the absolute URL to avoid the false negative for now =P

Reply:
Got it. Thanks for the clarification @ang-zeyu 👍